packages/astro/src/actions/runtime/virtual/shared.ts

```diff
@@ -309,3 +309,7 @@ const actionResultErrorStack = (function actionResultErrorStackFn() {
 		},
 	};
 })();
+
+export function astroCalledServerError(): AstroError {
+	return new AstroError(ActionCalledFromServerError);
+}
```

Comment:
Is this needed? AFAIK we usually directly throw an astro error without wrapping it in a function

Reply:
We do, however this particular function is used inside the template file `astro.mjs`, where we only import `astro:actions` virtual module.